_rules/role-attribute-valid-value-674b10.md

```diff
@@ -67,7 +67,7 @@ Element with multiple valid `role` values.
 Element with at least one valid `role` value.
 
 ```html
-<img role="img xyz" src="https://developer.mozilla.org/static/img/favicon144.png" alt="MDN logo" />
+<img role="img xyz" src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
```

Comment:
Agreed, but keep in mind this change needs a final call.

Reply:
Good thing I mentioned the Final Call in the PR description, then 😀